From 5986005375efd6338666b3f8a9ea021ce7a1eb4d Mon Sep 17 00:00:00 2001
From: Allen Xu <b45815@freescale.com>
Date: Thu, 25 Sep 2014 23:38:39 -0500
Subject: [PATCH 0482/1074] MLK-9654 ARM: dts: enable NAND support on i.MX6SX
 sabreauto board

enable NAND support on i.MX6SX sabreauto board

Signed-off-by: Allen Xu <b45815@freescale.com>
[Original patch taken from git://git.freescale.com/imx/linux-2.6-imx.git]
Signed-off-by: Biyao Zhai <biyao.zhai@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sabreauto.dts |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-sabreauto.dts b/arch/arm/boot/dts/imx6sx-sabreauto.dts
index 1287a44..7d311b4 100644
--- a/arch/arm/boot/dts/imx6sx-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6sx-sabreauto.dts
@@ -281,3 +281,9 @@
 		reg = <1>;
 	};
 };
+
+&gpmi {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_gpmi_nand_1>;
+	status = "okay";
+};
-- 
1.7.5.4

